tests/appveyor.prep.ps1

```diff
@@ -19,6 +19,12 @@ if (-not(Test-Path 'C:\Program Files\WindowsPowerShell\Modules\Pester\4.4.2')) {
     Install-Module -Name Pester -Force -SkipPublisherCheck -MaximumVersion 4.4.2 | Out-Null
 }
 
+#Setup DbatoolsConfig Path.DbatoolsExport path
+Write-Host -Object "appveyor.prep: Create Path.DbatoolsExport" -ForegroundColor DarkGreen
+if (-not(Test-Path 'C:\Users\appveyor\Documents\DbatoolsExport')) {
```

Comment:
A cleaner method might be to do this in each individual environment setup as the module is already loaded we won't have to hardcode the path. Meaning you simply can run `Get-DbatoolsConfig -Name Path.DbatoolsExport` to retrieve the path and validate in Appveyor it is there first.... Or you simply duplicate this process in the test themselves, which gives more context as to why it is there.

Reply:
so that would get moved to `tests\appveryor.SQL<ver>.ps1` instead of where it currently is?